tools/link.py

```diff
@@ -1349,6 +1349,9 @@ def phase_linker_setup(options, state, newargs):
   if '-lembind' in [x for _, x in state.link_flags]:
     settings.EMBIND = 1
 
+  if options.embind_emit_tsd or options.emit_tsd:
```

Comment:
Should we remove the `embind_emit_tsd` not that we have `emit_tsd`?   Or at least maybe just report as deprecated and ignore it?

Reply:
Yeah, I'll do that in another PR. I'll deprecate and document the new option.